build-system/tasks/build-story-localization.js

```diff
@@ -0,0 +1,110 @@
+const fs = require('fs-extra');
+const fastGlob = require('fast-glob');
+const pathMod = require('path');
+const debounce = require('../common/debounce');
+const {endBuildStep, watchDebounceDelay} = require('./helpers');
+const {watch} = require('chokidar');
+
+const dest = 'dist/v0';
+
+const LOCALES_DIR = 'extensions/amp-story/1.0/_locales/*.json';
+
+const FALLBACK_LANGUAGE_CODE = 'en';
+
+const LANGUAGE_CODE_CHUNK_REGEX = /\w+/gi;
+
+/**
+ * Finds fallback language codes for the current language code.
+ * This code is a copy of the same logic found in
+ * extensions/amp-story/1.0/amp-story-localization-service.js
```

Comment:
We will delete the code from here when this launches (because this will be handled in the backend), so idk if we need to mention the path here

Reply:
removed comment.